azuredevops/provider.go

```diff
@@ -23,48 +23,49 @@ import (
-			"azuredevops_serviceendpoint_azurerm":           serviceendpoint.ResourceServiceEndpointAzureRM(),
-			"azuredevops_serviceendpoint_bitbucket":         serviceendpoint.ResourceServiceEndpointBitBucket(),
-			"azuredevops_serviceendpoint_azuredevops":       serviceendpoint.ResourceServiceEndpointAzureDevOps(),
-			"azuredevops_serviceendpoint_dockerregistry":    serviceendpoint.ResourceServiceEndpointDockerRegistry(),
-			"azuredevops_serviceendpoint_azurecr":           serviceendpoint.ResourceServiceEndpointAzureCR(),
-			"azuredevops_serviceendpoint_github":            serviceendpoint.ResourceServiceEndpointGitHub(),
-			"azuredevops_serviceendpoint_github_enterprise": serviceendpoint.ResourceServiceEndpointGitHubEnterprise(),
-			"azuredevops_serviceendpoint_kubernetes":        serviceendpoint.ResourceServiceEndpointKubernetes(),
-			"azuredevops_serviceendpoint_runpipeline":       serviceendpoint.ResourceServiceEndpointRunPipeline(),
-			"azuredevops_serviceendpoint_sonarqube":         serviceendpoint.ResourceServiceEndpointSonarQube(),
-			"azuredevops_serviceendpoint_ssh":               serviceendpoint.ResourceServiceEndpointSSH(),
-			"azuredevops_serviceendpoint_npm":               serviceendpoint.ResourceServiceEndpointNpm(),
-			"azuredevops_serviceendpoint_genericwebhook":    serviceendpoint.ResourceServiceEndpointGenericWebhook(),
-			"azuredevops_git_repository":                    git.ResourceGitRepository(),
-			"azuredevops_user_entitlement":                  memberentitlementmanagement.ResourceUserEntitlement(),
-			"azuredevops_group_membership":                  graph.ResourceGroupMembership(),
-			"azuredevops_agent_pool":                        taskagent.ResourceAgentPool(),
-			"azuredevops_agent_queue":                       taskagent.ResourceAgentQueue(),
-			"azuredevops_group":                             graph.ResourceGroup(),
-			"azuredevops_project_permissions":               permissions.ResourceProjectPermissions(),
-			"azuredevops_git_permissions":                   permissions.ResourceGitPermissions(),
-			"azuredevops_workitemquery_permissions":         permissions.ResourceWorkItemQueryPermissions(),
-			"azuredevops_area_permissions":                  permissions.ResourceAreaPermissions(),
-			"azuredevops_iteration_permissions":             permissions.ResourceIterationPermissions(),
-			"azuredevops_build_definition_permissions":      permissions.ResourceBuildDefinitionPermissions(),
-			"azuredevops_check_invokerestapi":               resource.ResourceCheckInvokeRestAPI(),
-			"azuredevops_check_manualapproval":              manualapproval.ResourceCheckManualApproval(),
-			"azuredevops_build_permissions":                 permissions.ResourcePipelinePermissions(),
-			"azuredevops_serviceendpoint_githubapp":         githubapp.ResourceGithubApp(),
+			"azuredevops_resource_authorization":               build.ResourceResourceAuthorization(),
```

Comment:
If there's not some lint CI that enforces formatting, might be worth just leaving this line be awkwardly mis-indented, as it will reduce merge conflicts when updating from remote?